### PR TITLE
yum, whereis, whatis: add Persian translation

### PR DESCRIPTION
**Pull request by opoet7**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by marchersimon — APPROVED** [review]

**Comment by bl-ue** [comment]
This is odd...tldr-lint is commenting on a non-English PR, and it sent **two** messages. Also, I can't reproduce what it's saying.  Also the checks are green!!

@owenvoke do you know what's going on here?

File: pages.fa/linux/whatis.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# whatis
+
+> نمایش توضیحات یک خطی از صفحات راهنما.
```
Comment (navarroaxel) [inline pages.fa/linux/whatis.md:3]: google translate says `help pages`, do you have a Persian word for `manual pages` or this is fine?

Comment (opoet7) [reply pages.fa/linux/whatis.md:3]: I think صفحات راهنما best translation for manual pages.

**Comment by owenvoke** [comment]
@bl-ue, this isn't the linter posting, it's `markdown-lint` which is run across all pages and languages. 👍🏻

We currently use [these rules](https://github.com/tldr-pages/tldr/blob/master/.markdownlintrc), and it's run as part of [this script](https://github.com/tldr-pages/tldr/blob/7c681eea51f59c42391e0eacd3f4d91f19552bd7/scripts/test.sh#L13).

**Review by owenvoke — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]
Gotcha @owenvoke. 👍🏻

Thank you @opoet7!

File: pages.fa/linux/whatis.md (hunk 1 above)
Comment (bl-ue) [reply pages.fa/linux/whatis.md:3]: Looks like that's what's already there, so lgtm.

**Pull request merged**